backend/common/src/main/java/ai/verta/modeldb/common/monitoring/AuditLogInterceptor.java

```diff
@@ -62,7 +55,7 @@ public void close(Status status, Metadata trailers) {
                 } else {
                   message = String.format(AUDIT_WAS_CALLED_WRONG, callCount, methodName);
                 }
-                LOGGER.error(message);
+                //                LOGGER.error(message);
```

Comment:
sus?

Reply:
Removed bc it's extremely noisy on health checks. cc @ad-47 to make health checks not print these errors.

Reply:
oh yeah that one is annoying